### ISSUE TITLE
Wrong display of shopping points in the product page

### ISSUE THREAD

**Issue by ZioCan** [sarjon/gamifications#8]
Shopping points are calculated only if the product it's already in the cart, so in the product page always show 0 (zero) points.

**Comment by onlinedigitaltrade** [comment]
Yes that's also happening to me. I guess since it's a free module we'll have to fix it by ourselves.
